tests/test.py

```diff
@@ -579,10 +669,10 @@ def run_terraform(*, image: str, final: bool = False):
     # following files, and should have 4 logs lines in the fluent bit log
     # regardless of which image is being tested
     files = ["/tmp/kics_complete"]
+    files.append("/tmp/checkov_complete")
```

Comment:
I believe this was just a minor bug from a prior test refactor.  Since our base image has kics and checkov currently (checkov for terraform and kics for ansible) we are testing for both.  This test will break when we split terraform and ansible images out this test will fail, which is a good indication that the refactor is working.